oletools/msodde.py

```diff
@@ -777,19 +786,22 @@ def control_symbol(self, matchobject):
         # required to handle control symbols such as '\\'
         # inject the symbol as-is in the text:
         # TODO: handle special symbols properly
-        self.current_destination.data += matchobject.group()[1]
+        self.current_destination.data += matchobject.group()[1:2]
```

Comment:
I'm not sure about this change: what is the purpose?

Reply:
I had started testing msodde with the rtf spec and it failed in python3 saying:
  File "oletools/msodde.py", line 780, in control_symbol                                                                                                                                             
    self.current_destination.data += matchobject.group()[1]                                                                                                                                          
TypeError: can't concat bytes to int

The returned value is always just a single byte/char, had to convert it to bytes somehow. This did the trick but now that you point me to it, it is not very readable. The problem is, that bytes(int) just creates a 0-byte-array of given length. Maybe ... += bytes([matchobject.group()[1],]) or just bytes(matchobject.group()[1:]) is better?

Reply:
The test sample was added to tests/test-data/msodde/RTF-Spec-1.7.rtf in case you want to test yourself. Only happens with py3

Reply:
I see, then I need to handle python 2 and 3 cases differently. That code is just temporary anyway, because RTF control symbols need to be converted to bytes properly.
If your code works ok with python 2 and 3, I'll just merge it as-is for now.